contracts/MapleGlobals.sol

```diff
@@ -281,7 +285,7 @@ contract MapleGlobals {
         @param _fee The fee, e.g., 50 = 0.50%.
     */
     function setInvestorFee(uint256 _fee) external isGovernor {
-        _checkPercentageRange(_fee + treasuryFee);
+        _checkPercentageRange(treasuryFee.add(_fee));
```

Comment:
Good catch, and while obviously necessary, I wonder if this defeats the purpose of the original change to reduce cost/size compared to the original:
```suggestion
        _checkPercentageRange(_fee);
        _checkPercentageRange(_fee + treasuryFee);
```
In any case, this is obviously a needed change.